aws_lambda_powertools/utilities/idempotency/serialization/base.py

```diff
@@ -0,0 +1,23 @@
+"""
+Serialization for supporting idempotency
+"""
+import logging
+from abc import ABC, abstractmethod
+from typing import Any, Dict
+
+logger = logging.getLogger(__name__)
+
+
+class BaseDictSerializer(ABC):
+    """
+    Abstract Base Class for Idempotency serialization layer, supporting dict operations.
+    This interface not be inherited by end user implementation.
```

Comment:
Can you rephrase this sentence? I'm not sure what you mean here. maybe we're missing a "should"?

```suggestion
    This interface should not be inherited by end user implementation.
```

Reply:
removed the line alltogehter